tools/cordova/builder.js

```diff
@@ -248,7 +248,7 @@ export class CordovaBuilder {
   writeConfigXmlAndCopyResources(shouldCopyResources = true) {
     const { XmlBuilder } = loadIsopackage('xmlbuilder');
```

Comment:
Since this is basically the only user of the xmlbuilder package in Meteor I wonder why we don't load the npm package directly. @benjamn @filipenevola : Can you shine some light on this?

Reply:
I think that makes alot of sense, I didnt want to deviate too far from the way it was I suppose. But that was the only file making use of the xmlbuilder package. I could update this branch and remove the xml builder package if you guys decide that's a better approach

Reply:
I made a branch where the xmlbuilder isopack (i can see why this name was needed now haha) has been removed, and then in the builder.js file the xmlbuilder2 node package is imported directly. As a work around i installed the xmlbuilder2 library using npm in the tools directory's node_modules folder so I could do some testing. But I dont know how to add a new node module to the what the startup script calls the "dependency kit".

Reply:
The node_modules of the dependency kit are defined in: `./scripts/dev-bundle-tool-package.js`

If you change these, you should also change the dev bundle version in `./meteor` .

The consequence is that the tests here will fail, since the bundle first needs to be published.

However, there might still be a reason I'm not aware of this xmlbuilder was loaded from an isopackage instead of NPM. So maybe it's better to wait for feedback from @benjamn if this is the right approach.

Reply:
thats sounds good to me, whatever direction you guys decide is best. I was preparing for both scenarios and was curious how to make updates to the tools area of the project as well. Thanks for letting me know how to update the dependency kit too, Ill play around with this branch locally in the mean time :)

Reply:
I am 100% in favor of adding `xmlbuilder2` to the dev bundle, rather than loading it into the command-line tools codebase from a Meteor package. The implementation of that package is just one line, after all:
```
XmlBuilder = Npm.require('xmlbuilder');
```
Maybe we can also move `meteor/packages/xmlbuilder` into `meteor/packages/non-core` while we're at it, since that package is definitely not used by any core packages (including `meteor-tool`) now?

Reply:
I moved the original version of the xmlbuilder isopack to the non-core folder, and then added the xmlbuilder2 node package to the dev bundle. The license file now has both libraries included too. I tested this branch by creating a config.xml and comparing it against one created using the 1.9 release and they are identical. The only thing I am not sure of it the dev bundle version I chose. The most recent one I noticed in the repo was 12.16.1.3 so i bumped it to 12.17.1.3, but the xmlbuider api is changing so 13.0.0.0 might make sense too

Reply:
I should have mentioned that the dev bundle version doesn't follow semver. Basically we keep the first numbers for the node version (so that should stay 12.16.1) and then just increase the last number for other changes that need a new dev bundle, so `12.16.1.4` is ok.

Reply:
gotcha .. let me make that change :)